src/OpenTelemetry.Instrumentation.SqlClient/Implementation/SqlEventSourceListener.netfx.cs

```diff
@@ -196,6 +205,29 @@ private void OnEndExecute(EventWrittenEventArgs eventData)
                 activity.Stop();
             }
         }
+
+        private void RecordException(EventSource eventSource, Activity activity, string errorText)
+        {
+            var tagsCollection = new ActivityTagsCollection
+            {
+                // The real exception object is unavailable via the EventSource but we
+                // know that it must be of type "SqlException".
+                {
+                    SemanticConventions.AttributeExceptionType,
+
+                    // The new "Microsoft.Data.SqlClient.EventSource" for sure corresponds to "Microsoft.Data.SqlClient.SqlException".
+                    // The old "Microsoft-AdoNet-SystemData" could be either type, but most likely "System.Data.SqlClient.SqlException".
+                    eventSource == this.mdsEventSource ? MdsExceptionName : AdoNetExceptionName
```

Comment:
Given the fact that we cannot get actual exception in .NET Framework, what If we make `RecordException` only available in .NETCore?
In .NET Framework case, we are not adding much value by reporting the  (known) Exceptiontype and the errorcode, as errorcode is already available inside Status.

Reply:
Ah, that's a good idea - thanks! Will do.

Reply:
Okay updated now, sorry for delay! Also fixed up CHANGELOG and README.